frontend/components/CompanyListHeader.vue

```diff
@@ -0,0 +1,31 @@
+<template>
+  <div
+    class="px-2 py-1 pt-4 pb-5 rounded-lg border-b bg-gray-50 border-gray-200"
+  >
+    <h3 class="flex justify-center text-lg leading-6 font-medium text-gray-900">
+      Lista firm
+    </h3>
+    <p class="mt-1 max-w-4xl text-sm text-gray-500">Znaleziono: 0</p>
+  </div>
+  <div class="w-full">
+    <label for="search" class="sr-only">Wyszukaj</label>
+    <div class="relative">
+      <div
+        class="pointer-events-none absolute inset-y-0 left-0 pl-3 flex items-center"
+      >
+        <SearchIcon class="h-5 w-5 text-gray-400" aria-hidden="true" />
+      </div>
+      <input
```

Comment:
Change color of border, because it is black or I'm just blind.

Reply:
Color of border was change to brighter